pkg/build/controller/build/build_controller.go

```diff
@@ -124,6 +127,25 @@ type registries struct {
 
 type tomlConfig struct {
 	Registries registries `toml:"registries"`
+}*/
+
+type endpoint struct {
```

Comment:
Why not import the actual types from GitHub.com/containers/image, now that they have been made public for MCO doing this?

Reply:
Yep I looked into that but that would require a vendor/deps bump, and the github.com/containers/image vendored into this repo does not have those setting yet.

I did not want to take that on yet with a WIP / is this even the right path sort of initial change.

I should have left a TODO explaining all that.  I'll do so on the next bump.

Reply:
Of course the switch to https://github.com/openshift/machine-config-operator/blob/cfb78728a6e985fc40a3ef318ba1de359de17a6d/pkg/controller/container-runtime-config/helpers.go#L321 might mitigate that ... I see it is using toml